app/modules/individuals/resources.py

```diff
@@ -18,12 +19,35 @@
 
 from . import parameters, schemas
 from .models import Individual
-
+from app.modules.encounters.models import Encounter
 
 log = logging.getLogger(__name__)  # pylint: disable=invalid-name
 api = Namespace('individuals', description='Individuals')  # pylint: disable=invalid-name
 
 
+def _cleanup_post_and_abort(db, guid, message='Unknown error'):
```

Comment:
For the Sightings this is now done as a class object which is cleaner. 
I think this is copied from the previous sightings cleanup logic
https://github.com/WildMeOrg/houston/blob/main/app/modules/sightings/resources.py#L33

Reply:
I'll try to follow the pattern, but I'm not convinced it's cleaner for Individual. What was the motivation to change the approach for Sighting?